### DISCUSSION TITLE
several inverters in parallel mode

### DISCUSSION THREAD

**Discussion by Schlumpf53** [kellerza/sunsynk#200] [Q&A] [answered]
Hi. im running two and there is going to be more Deye inverters in parallel mode and i want to use this addon to get the data via modbus rs485. I´m already using it succesfully with one inverter, but i dont know how to use it with several.
Does anyone have an idea or documentation regarding this?
Thank you.

**Comment by purcell-lab** [comment, marked as answer]
I just added additional inverter descriptions into the configuration and both devices have now come up.

![image](https://github.com/kellerza/sunsynk/assets/79175134/9a3281ba-c1ee-493b-8304-f2efba7f7ca8)

**Reply by kellerza** [reply]
One note, with RS485 you will likely need one usb-rs485 adaptor per inverter (even if the modbus IDs don’t overlap)
